<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomConfigDialog</class>
 <widget class="QDialog" name="RoomConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Room Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_UI">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="tilesetLabel">
         <property name="text">
          <string>Tileset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_TilesetID"/>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_BGLayerEnable">
         <property name="text">
          <string>Enable Background Layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="backgroundLabel">
         <property name="text">
          <string>Use existing Background Layer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_BGLayerPicker">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_BGLayerAutoScroll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Background AutoScroll Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0Enable">
         <property name="text">
          <string>Enable Layer 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0Alpha">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Layer 0 Alpha Enable:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="alphaBlendingLabel">
         <property name="text">
          <string>Alhpa Blend Attributes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_AlphaBlendAttribute">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L0MappingTypeLabel">
         <property name="text">
          <string>Layer 0 Mapping Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_Layer0MappingType">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L0Label">
         <property name="text">
          <string>Use existing Layer 0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_Layer0Picker">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0AutoScroll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Layer 0 AutoScroll Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer2Enable">
         <property name="text">
          <string>Enable Layer 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="layerPriorityLabel">
         <property name="text">
          <string>Layer Priority:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_LayerPriority"/>
       </item>
       <item>
        <widget class="QLabel" name="roomWidthLabel">
         <property name="text">
          <string>Room Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_RoomWidth">
         <property name="minimum">
          <number>19</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="roomHeightLabel">
         <property name="text">
          <string>Room Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_RoomHeight">
         <property name="minimum">
          <number>14</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_View">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="graphicViewDetailsLabel">
         <property name="text">
          <string>Tileset: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RoomPreviewGraphicsView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to see other detail(s)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoomPreviewGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>Dialog/RoomPreviewGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RoomConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RoomConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
